### ISSUE TITLE
Eurofighter Typhoon Air-2-Ground Loadouts

### ISSUE THREAD

**Issue by Quark-73** [PygmalionOfCyprus/cmo-db-requests#3946]
### DB Selector

DB3K

### Affected DBID(s)

Loadout, 19159, 19160, 2748, 2746, 31012, 1336, 2192, 17285, 17288,17286, 17287

### Summary of Changes

Hello,
I am not sure if it is intended for some reason, but it seems to me that there is a artificial limitation of the Eurofighters A2G loadouts to 2 AMRAAMs/Meteors.
The Eurofighter has 13 hardpoints to attach payload. The tip stations that carry SRAAMs only. It has also three pairs of underwing pylons. The outboard stations can carry, SRAAM, MRAAM, BVRAAM, (light weight) bombs and (light weight) missiles. The centre wing stations can optionally carry underwing tanks. The inboard stations The outboard stations can carry,  MRAAM, BVRAAM, (heavy weight) bombs and (heavy weight) missiles. SRAAM is possible but I've seen that on this station.
Then it has four fuselage stations for either AMRAAM or Meteor. These four stations don't stay in concurrency to any other stations. That means whatever you put on the aircraft you have always room for four MRAAM/BVRAAM. Number thirteen is the centre line station for either drop tank or laser designator pod.  
I can't give any information about additionally radar cross section of 4 instead of two missiles, but should be negletable or much less than on normal pyplon since the under fuselage stations are conformal with the fuselage (see picture).

I would suggest to adapt at least the A2S loadouts to 4 MRAAM/BVRAAM. Basically this applies to all EF2000 operators beside Austria.
But if I look on the Eurofighter loadouts in general there is none I found that is utilising all 13 of the hardpoints.

KInd Regards
Quark

### Sources

https://en.wikipedia.org/wiki/Eurofighter_Typhoon#Armament

https://en.wikipedia.org/wiki/Eurofighter_Typhoon#/media/File:Eurofighter_Typhoon_FGR.4_'ZK356'_(35359259191).jpg

**Comment by FrangibleCover** [comment]
I agree with this, all loadouts should fill the recessed missile stations with missiles. Typhoon has long been marketed as a 'swing role' aircraft intended to carry munitions suitable for engaging air and ground targets in the same mission (like an F-4B fifty years before it). It's literally the first thing they talk about on the website: https://www.eurofighter.com/the-aircraft

Whether this precisely agrees with doctrine or how the aircraft would be operated in practice is difficult to say, but I reckon it's probably right.

**Comment by bostonmyk** [comment]
Here you go:

https://x.com/supersonicessna/status/1616087686900600835/photo/1
